html/cross-origin-opener-policy/resources/coop-coep.py

```diff
@@ -21,13 +21,41 @@ def main(request, response):
 <script src="/common/get-host-info.sub.js"></script>
 <iframe></iframe>
 <script>
-  const navigate = new URL(location).searchParams.get("navigate");
-  if (navigate !== null) {
-    self.location = navigate;
+  const params = new URL(location).searchParams;
+  const navHistory = params.get("navHistory");
+  const avoidBackAndForth = params.get("avoidBackAndForth");
+  const navigate = params.get("navigate");
+  const openerDOMAccess = params.get("openerDOMAccess");
```

Comment:
This constant is unused.

Reply:
I'm not sure why I thought I needed this. Removed. Though I'd like to check now how this impacts other tests...

Reply:
For now, I've skipped the assertion when `openerDOMAccess` is undefined.